src/EditorFeatures/VisualBasicTest/Diagnostics/AbstractVisualBasicDiagnosticProviderBasedUserDiagnosticTest.vb

```diff
@@ -43,5 +44,41 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.Diagnostics
         Protected Overrides Function GetLanguage() As String
             Return LanguageNames.VisualBasic
         End Function
+
+        Friend ReadOnly Property RequireArithmeticBinaryParenthesesForClarity As IOptionsCollection
```

Comment:
1. feels like this is shareable with C#.  
2. you can write this as `Friend ReadOnly Property RequireArithmeticBinaryParenthesesForClarity As IOptionsCollection = ParenthesesOptionsProvider.RequireArithmeticBinaryParenthesesForClarity`.  One line instead of 5 :)

Reply:
Sure, I will clean this up in the next PR.